pkg/services/folder/folderimpl/folder.go

```diff
@@ -922,6 +1276,51 @@ func (s *Service) legacyDelete(ctx context.Context, cmd *folder.DeleteFolderComm
 }
 
 func (s *Service) Move(ctx context.Context, cmd *folder.MoveFolderCommand) (*folder.Folder, error) {
+	if s.features.IsEnabledGlobally(featuremgmt.FlagKubernetesFoldersServiceV2) {
```

Comment:
Can we in this case use `FlagK8SFolderMove` instead? This endpoint currently has a slightly different behaviour than the legacy one, so I'd like to keep it under a different feature toggle.